app/secure/index.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
```

Comment:
@maxcbc is this the right place or should it be in utils or middleware?

Reply:
In cc frontend is not a middleware, it is a normal endpoint. Let's wait for @maxcbc  :)

Reply:
normal endpoint by the looks of it.